src/Symfony/Component/OptionsResolver/Tests/OptionsResolverTest.php

```diff
@@ -36,6 +36,35 @@ protected function setUp(): void
         $this->resolver = new OptionsResolver();
     }
 
+    public function provideResolveWithIgnoreUndefined()
```

Comment:
should be made static + be put after the test case

Reply:
Please add a return type